multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -138,15 +138,20 @@ def parse_coverage(self, f):
         log.debug("Couldn't parse contents of coverage histogram file {}".format(f['fn']))
         return None
 
-    # Find median without importing anything to do it for us
+    # Find median and mean without importing anything to do it for us
     num_counts = sum(d.values())
     cum_counts = 0
+    total_cov = 0
     median_coverage = None
+    mean_coverage = None
```

Comment:
I don't think you need this variable? You can just assign directly to `self.general_stats_data[s_name]['mean_coverage']` at the end and save two lines of code..